experiments/experiment_prototype.py

```diff
@@ -1,385 +1,1111 @@
-            followed by cpo #2 and subsequent cpo's.
-        INTTIME : nave by nave interfacing (full integration time of
-             one sequence, then the next). Time/number of sequences
-            dependent on intt and intn in cp_object. Effectively
-            simultaneous scan interfacing, interleaving each
-            integration time in the scans. cpo #1 first inttime or
-            beam direction will run followed by cpo #2's first inttime,
-            etc. if cpo #1's len(scan) is greater than cpo #2's, cpo
-            #2's last integration will run and then all the rest of cpo
-            #1's will continue until the full scan is over. CPObject 1
-            and 2 must have the same scan boundary, if any boundary.
-            All other may differ.
-        INTEGRATION : integration by integration interfacing (one
-            sequence of one cp_object, then the next). CPObject #1 and
-            CPO #2 must have same intt and intn. Integrations will
-            switch between one and the other until time is up/nave is
-            reached.
-        PULSE : Simultaneous sequence interfacing, pulse by pulse
-            creates a single sequence. CPO A and B might have different
-            frequencies (stereo) and/or may have different pulse
-            length, mpinc, sequence, but must have the same integration
-            time. They must also have same len(scan), although they may
-            use different directions in scan. They must have the same
-            scan boundary if any. A time offset between the pulses
-            starting may be set (seq_timer in cp_object). CPObject A
-            and B will have integrations that run at the same time.
-        """
-
-    def __call__(self): # REVIEW #28 It seems this method is used to print stuff. try to use __repr__ or __str__ instead http://stackoverflow.com/questions/1436703/difference-between-str-and-repr-in-python
-        print 'CPID [cpid]: {}'.format(self.cpid)
-        print 'Num of CP Objects [cponum]: {}'.format(self.cponum)
-        for i in range(self.cponum):
-            print '\n'
-            print 'CP Object : {}'.format(i)
-            print self.cpo_list[i]
-        print '\n'
-        print 'Interfacing [interface]: {}'.format(self.interface)
-        return None
+        self._interface = {}  # setup_interfacing(self.num_slices)
+        # Dictionary of how each exp_slice interacts with the other slices. Default is "NONE" for
+        #  all, but must be modified in experiment. NOTE keys are as such: (0,1), (0,2), (1,2),
+        # NEVER includes (2,0) etc. The only interface options are: interface_types = frozenset([
+        # 'SCAN', 'INTTIME', 'INTEGRATION', 'PULSE'])
+
+        # The following are for internal use only, and should not be modified in the experimental
+        #  class, but will be modified by the class methods to build_scans. For this reason they
+        # are private with setters.
```

Comment:
Okay we only see a getter for __scan_objects, and nothing for __slice_id_scan_lists. Does this comment need an update?